config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,6 @@
+# Be sure to restart your server when you modify this file.
+
+# ApplicationController.renderer.defaults.merge!(
+#   http_host: 'example.org',
+#   https: false
+# )
```

Comment:
I'll have a look at what this file is for in a bit, but it might be worth deleting it if it is unnecessary?